portal-impl/src/com/liferay/portal/service/impl/CompanyServiceImpl.java

```diff
@@ -87,7 +87,7 @@ public Company addCompany(
 	@JSONWebService(mode = JSONWebServiceMode.IGNORE)
 	@Override
 	public Company addCompany(
-			String webId, String virtualHost, String mx, int maxUsers,
+			Long companyId, String webId, String virtualHost, String mx, int maxUsers,
```

Comment:
@javalosjr96 The parameter is added but not used. By the way you may need to run baseline and if the major version is bumped, document a breaking change.